src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,14 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
-);
+export const MovieList = ({ movies }) => {
+  const moviesList = movies.map(movie => (
+    <MovieCard movie={movie} key={movie.imdbId} />
+  ));
+
+  return (
+    <div className="movies">
+      { moviesList }
+    </div>
+  );
```

Comment:
I think you know that you can directly return iteration, without creating new variable, but it useful when you need use it in different places.
```suggestion
  return (
    <div className="movies">
      {movies.map(movie => (
      <MovieCard movie={movie} key={movie.imdbId} />
    ))}
    </div>
  );
```